### ISSUE TITLE
NTR: Glycogen brancher enzyme activity low in skin fibroblast (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9113]
New term request Glycogen brancher enzyme activity low in skin fibroblast (segal_090511164419): GBE; Amylo(1,4 - 1,6) transglucosidase deficiency  UMLS: 1856314

**Comment by pnrobinson** [comment]
@MickeySegal
do we mean Glycogen branching (not brancher) enzyme?
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4581599/

**Comment by MickeySegal** [comment]
Both names have been in use, but branching seems to be the preferred form now.

I switched in our term name and included brancher as a synonym.

**Comment by pnrobinson** [comment]
Looking here, this phenotype can be measured in many tissues including blood.
https://omim.org/entry/232500
Fibrblasts do not seem to be the most relevant tissue pathophysiologically. Maybe a blood/general term would do it?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK115333/ "The diagnosis is confirmed by the demonstration of glycogen branching enzyme (GBE) deficiency in liver, muscle, or skin fibroblasts [Brown & Brown 1983], and/or the identification of biallelic pathogenic variants in GBE1."

So fibroblasts do seem relevant, but a more general term would be fine.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000861
- label: Diminished tissue glycogen branching enzyme activity
- parents: [Abnormal tissue enzyme concentration or activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:26199317
- def: Activity of glycogen branching enzyme in tissues below the lower limit of normal. The activity can be measured in multiple tissues including culutured fibroblasts.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9113
- synonyms:
